src/controller/CHIPDeviceController.cpp

```diff
@@ -3424,6 +3443,26 @@ void DeviceCommissioner::PerformCommissioningStep(DeviceProxy * proxy, Commissio
         );
     }
     break;
+    case CommissioningStage::kPrimaryOperationalNetworkFailed: {
+        // nothing to do. This stage indicates that the primary operation network failed and the network interface should be
+        // disabled later.
+        break;
+    }
+    case CommissioningStage::kDisablePrimaryNetworkInterface: {
+        NetworkCommissioning::Attributes::InterfaceEnabled::TypeInfo::Type request = false;
+        CHIP_ERROR err                                                             = SendCommissioningWriteRequest(
+            proxy, endpoint, NetworkCommissioning::Attributes::InterfaceEnabled::TypeInfo::GetClusterId(),
+            NetworkCommissioning::Attributes::InterfaceEnabled::TypeInfo::GetAttributeId(), request,
```

Comment:
Why not just write `NetworkCommissioning::Id` and `NetworkCommissioning::Attributes::InterfaceEnabled::Id`?  Would be shorter.  If you were templating on the typeinfo it would make sense to use the getters on it, but you're not.

Reply:
Done.